packages/libs/react-ui/src/components/Accordion/Accordion.css.ts

```diff
@@ -53,8 +56,20 @@ export const accordionContentClass = style([
   sprinkles({
     color: '$neutral5',
     fontSize: '$base',
+    margin: 0,
     overflow: 'hidden',
+    padding: 0,
     paddingBottom: '$2',
-    paddingTop: '$2',
+  }),
+]);
+
+export const accordionCollapse = style([
```

Comment:
This is in preparation for [animating the opening and closing](https://app.asana.com/0/1205693948735844/1205693670996249/f). Rather than only rendering the Accordion content when a section is open, we need to have the content immediately available so that we can calculate its height. For now, we just switch between display `block` and `none` to show and hide.

Reply:
Alright, let's tackle this problem in the task for it then. I'll remove this for now.